spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/web/TraceFilter.java

```diff
@@ -202,6 +193,19 @@ else if (skip) {
 		}
 	}
 
+	private void logServerReceived(Span parent) {
+		if (parent != null && parent.isRemote()) {
+			parent.logEvent(Span.SERVER_RECV);
+		}
+	}
+
+	private void logServerSent(SpanReporter spanReporter, Span parent) {
```

Comment:
I think these two methods are better inlined. For example, this method conditionally logs server sent, and also reports! The extraction doesn't help line count that much and the method name hides some pretty important details.